install/ci-vm/ci-linux/ci/runCI

```diff
@@ -57,6 +103,15 @@ postStatus "preparation" "Copy code to local folder"
 executeCommand cp -r ${srcDir}/* ${dstDir}
 executeCommand cd ${dstDir}
 
+postStatus "cmake building" "Compiling CCExtractor using cmake"
```

Comment:
What if any of these fails? We would like to run the tests with the "regular" built version regardless of build failures on one of these three...